### ISSUE TITLE
Unable to use mommy with Models which have an attribute `model`

### ISSUE THREAD

**Issue by jdunck** [vandersonmota/model_mommy#333]
We have a django `Model` which has an attribute, `model`.  When attempting `mommy.make` with it:

```
  File "/home/vagrant/venvs/mp_web/lib/python2.7/site-packages/model_mommy/recipe.py", line 68, in make
    return mommy.make(self.model, **self._mapping(attrs))
TypeError: make() got multiple values for keyword argument 'model'
```

There are a couple solutions I can imagine:

1) internally, `mommy` could call the variable `model` something else.  
2) shift from accepting `**kwargs` to accepting `attrs={}`.

The 2nd option is unappealing because it would make the interface more clunky (and also be an upgrade path issue for library clients).

What do you think of the 1st option?  I'd be happy to send up a PR if you have a variable name you'd prefer.

**Comment by berinhard** [comment]
Thanks for the issue @jdunck! I guess we could follow with your first suggestion. I think we should follow the same patter we already do with `_quantity`, for example, and rename it to `_model`.

**Comment by jdunck** [comment]
OK, PR is up.  Let me know if any changes are needed?

**Comment by vandersonmota** [comment]
fixed on #335
